components/style/animation.rs

```diff
@@ -494,6 +494,8 @@ fn compute_style_for_animation_step(context: &SharedStyleContext,
 
             // This currently ignores visited styles, which seems acceptable,
             // as existing browsers don't appear to animate visited styles.
+            let registered_property_set =
+                context.stylist.registered_property_set().read_with(context.guards.registered_property_set);
```

Comment:
Ugh, I didn't remember we had `context.stylist` down here. Looks fine I guess, though it seems prone to abuse.

So far the only two callers are these too, so it's fine.